pkg/fuzzy/googletranslate.go

```diff
@@ -0,0 +1,137 @@
+package fuzzy
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"cloud.google.com/go/translate"
+	"github.com/spf13/viper"
+	"go.expect.digital/translate/pkg/model"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"golang.org/x/text/language"
+	"google.golang.org/api/option"
+	htransport "google.golang.org/api/transport/http"
+)
+
+// --------------------Definitions--------------------
+
+// Interface that defines some of the methods of the Google Translate client.
+// This interface helps to mock the Google Translate client in unit tests.
+// https://pkg.go.dev/cloud.google.com/go/translate#Client
+type googleClient interface {
+	Translate(ctx context.Context, inputs []string, target language.Tag, opts *translate.Options) ([]translate.Translation, error) //nolint:lll
+	io.Closer
+}
+
+// googleTranslate implements the TranslationService interface.
+type googleTranslate struct {
```

Comment:
NewGoogleTranslate returns this struct - always return an exposed identifier in the exposed function.

```suggestion
type GoogleTranslate struct {
```